src/NCalc/Expression.cs

```diff
@@ -228,7 +229,15 @@ public bool HasErrors()
         protected Dictionary<string, IEnumerator> ParameterEnumerators;
         protected Dictionary<string, object> ParametersBackup;
 
-        public Func<TResult> ToLambda<TResult>()
+        private struct Void { };
+
+        public struct ExpressionWithParameter
+        {
+            public L.Expression expr;

```

Comment:
should these public fields be pascal-case to match conventions?

Reply:
+

Reply:
\+